pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: chunked_uploader
 description: An easy-to-use package to upload files in chunks using Dio for mobile, desktop and web.
-version: 1.1.0
+version: 1.2.1
```

Comment:
Can you please revert this? I will change it later when preparing the release.

Reply:
Yep! Will do it ASAP!